pkg/encoding/lorawan/lorawan_test.go

```diff
@@ -818,3 +819,675 @@ func TestLoRaWANEncodingRawJoinAcceptPayload(t *testing.T) {
 		})
 	}
 }
+
+func TestDeviceEIRPToFloat32(t *testing.T) {
+	for _, tc := range []struct {
+		Enum  ttnpb.DeviceEIRP
+		Float float32
+	}{
+		{
```

Comment:
It may be nice to format these on one line each so that they look a little more like a table